packages/connectivity/connectivity/android/src/main/java/io/flutter/plugins/connectivity/Connectivity.java

```diff
@@ -103,4 +118,38 @@ private String getNetworkTypeLegacy() {
   public ConnectivityManager getConnectivityManager() {
     return connectivityManager;
   }
+
+  private Boolean checkRequirement() {
+    if (Build.VERSION.SDK_INT < Build.VERSION_CODES.P) return true;
```

Comment:
nit: avoid having `return` on the same line, and use `{}`
```suggestion
    if (Build.VERSION.SDK_INT < Build.VERSION_CODES.P) {
      return true;
    }
```